pytorch_lightning/trainer/trainer.py

```diff
@@ -316,8 +316,61 @@ def __init__(
             # https://github.com/PyTorchLightning/pytorch-lightning/pull/1572/files#r420279383
             os.environ["HOROVOD_FUSION_THRESHOLD"] = str(0)
 
-        # Init callbacks
+        # training bookeeping
+        self.total_batch_idx = 0
+        self.running_loss = TensorRunningAccum(window_length=20)
+        self.batch_idx = 0
+        self.progress_bar_metrics = {}
+        self.callback_metrics = {}
+        self.num_val_batches = 0
+        self.num_training_batches = 0
+        self.num_test_batches = 0
+        self.train_dataloader = None
+        self.test_dataloaders = None
+        self.val_dataloaders = None
+
+        # training state
+        self.model = None
+        self.testing = False
+        self.disable_validation = False
+        self.lr_schedulers = []
+        self.optimizers = None
+        self.optimizer_frequencies = []
+        self.global_step = 0
+        self.current_epoch = 0
+        self.interrupted = False
+        self.should_stop = False
+
+        # set default save path if user didn't provide one
+        if default_root_dir is None or default_root_dir is True:
+            # Backward compatibility, TODO: remove default_save_path in v0.8.0
```

Comment:
```suggestion
            # Backward compatibility, TODO: remove default_save_path in v0.8.0
            rank_zero_warn("Argument `default_root_dir` has renamed to `num_nodes` since v0.5.x"
                           " and this method will be removed in v0.8.0", DeprecationWarning)
```

Reply:
this warning is incorrect, it should state "Argument `default_save_path ` has renamed to `default_root_dir `"